.github/workflows/python.yml

```diff
@@ -0,0 +1,33 @@
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name:  Python Package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3, 3.5, 3.6]
```

Comment:
change this to include more versions please: https://endoflife.date/python

Reply:
i have added other updated versions